packages/components/src/Button/Button.css

```diff
@@ -159,21 +134,17 @@ a.button:active {
   background-color: var(--button--color-surface--hover);
 }
 
-.tertiary:hover,
-.tertiary:focus-visible,
-a.tertiary:hover,
-a.tertiary:focus-visible,
 .subtle.secondary:hover,
 .subtle.secondary:focus-visible,
 a.subtle.secondary:hover,
 a.subtle.secondary:focus-visible {
   border-color: transparent;
 }
 
-:global(.jobber-retheme) .tertiary:hover,
-:global(.jobber-retheme) .tertiary:focus-visible,
-:global(.jobber-retheme) a.tertiary:hover,
-:global(.jobber-retheme) a.tertiary:focus-visible {
+.tertiary:hover,
+.tertiary:focus-visible,
+a.tertiary:hover,
+a.tertiary:focus-visible {
```

Comment:
@fagnanm do you want to revisit the "tertiary also have borders" decision in the scope of this? I could simplify this by grouping these selectors with the `secondary` styles defined above

Reply:
@chris-at-jobber yes, I wrote a ticket up here for it https://jobber.atlassian.net/browse/JOB-92981